packages/ui/src/components/Bullet/index.tsx

```diff
@@ -88,8 +89,9 @@ export const Bullet = ({
   text,
   tooltip,
   tooltipBaseId,
+  'data-testid': dataTestId,
 }: BulletProps) => (
-  <Tooltip id={tooltipBaseId} text={tooltip}>
+  <Tooltip id={tooltipBaseId} text={tooltip} data-testid={dataTestId}>
     <StyledContainer variant={variant} size={size} className={className}>
```

Comment:
It should not be applied on tooltip but on the container instead:

```suggestion
  <Tooltip id={tooltipBaseId} text={tooltip}>
    <StyledContainer variant={variant} size={size} className={className} data-testid={dataTestId}>
```